mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -32,19 +34,30 @@ func init() {
 	register.RegisterTest(&register.Test{
 		Name:        "coreos.ignition.failure",
 		Description: "Verify ignition will fail with unsupported action.",
-		Run:         runIgnitionFailure,
+		Run:         runIgnitionTestGroup,
```

Comment:
Rather than growing the same test, let's make it a separate test with its own more specific name. I.e. call `register.RegisterTest()` again with a separate `register.Test{}` struct.

Reply:
Latest update considers above:
https://github.com/coreos/coreos-assembler/pull/3647/commits/9376cceb2930966fff0cade45d3292fb669b88c7#diff-3ad99cf50017bf15cd0eae37f7571c6e217480c1c2b1748266bc7eb417cb0aacR38-R46